hypervisor/virt/hyperv/hypervcli.py

```diff
@@ -261,3 +261,45 @@ def guest_resume(self, guest_name):
         else:
             logger.error("Failed to resume hyperv guest")
             return False
+
+            
+    def hypervisor_guid(self):
```

Comment:
we already have this function, see https://github.com/VirtwhoQE/hypervisor-builder/blob/master/hypervisor/virt/hyperv/hypervcli.py#L96

Reply:
OK. I see